src/main/scala/com/yotpo/metorikku/code/steps/obfuscate/ObfuscateColumns.scala

```diff
@@ -0,0 +1,49 @@
+package com.yotpo.metorikku.code.steps.obfuscate
+
+import org.apache.spark.sql.{Column, DataFrame}
+import org.apache.spark.sql.functions.{col, lit, udf, when}
+import com.roundeights.hasher.Implicits._
+import com.yotpo.metorikku.code.steps.InputMatcher
+import com.yotpo.metorikku.exceptions.MetorikkuException
+
+object ObfuscateColumns {
+  private val ObfuscateColumnsInputMatcher = InputMatcher("columns", "delimiter", "value", "table")
+
+  def run(ss: org.apache.spark.sql.SparkSession, metricName: String, dataFrameName: String, params: Option[Map[String, String]]): Unit = {
+    val (columns, value, table) = parse(params)
+    val df = ss.table(table)
+    val newDataFrame = obfuscateColumns(df, columns, value)
+
+    newDataFrame.createOrReplaceTempView(dataFrameName)
+  }
+
+  private def parse(params: Option[Map[String, String]]): (Array[String], String, String) = {
+    params.get match {
+      case ObfuscateColumnsInputMatcher(columns, delimiter, value, table) => (columns.split(delimiter), value, table)
+      case _ => throw MetorikkuException(
+        """Obfuscate Columns expects the following parameters: columns,
+          delimiter, value and table. one or more of those wasn't supplied""".stripMargin
```

Comment:
Pandas are advocates of the Oxford comma!

Reply:
https://youtu.be/j64SctPKmqk?t=92